notebooks/utils/notebook_utils.py

```diff
@@ -623,9 +623,9 @@ def show_live_inference(ie, image_paths: List, model: model.Model, device: str):
 
 def benchmark_model(model_path: os.PathLike,
```

Comment:
```suggestion
def benchmark_model(model_path: PathLike,
```
Also changed in ipynb file, fix inconsistency in existing code.